Jenkinsfile

```diff
@@ -30,7 +30,7 @@ pipeline {
 					
 					mvn clean install -f org.eclipse.jdt.core.compiler.batch -DlocalEcjVersion=99.99 -Dmaven.repo.local=$WORKSPACE/.m2/repository -DcompilerBaselineMode=disable -DcompilerBaselineReplace=none
 					
-					mvn -U clean verify --batch-mode --fail-at-end -Dmaven.repo.local=$WORKSPACE/.m2/repository \
+					mvn -U clean verify --batch-mode --fail-at-end -X -Dmaven.repo.local=$WORKSPACE/.m2/repository \
 						-Ptest-on-javase-24 -Pbree-libs -Papi-check -Pjavadoc -Pp2-repo \
 						-Dmaven.test.failure.ignore=true \
 						-Dcompare-version-with-baselines.skip=false \
```

Comment:
```suggestion
						-Dcompare-version-with-baselines.skip=true \
```

You can try this if you just don't want to see the error (with possible other obstacles then), but this only hides the problem that the BETA_24 branch seems to be incorrectly configured (either with respect to the used baseline repo or git-parent, or ...) or your PR needs some changes.

For example if I look here it says the beta-branch is behind master (but that might be intended I don't know):

![grafik](https://github.com/user-attachments/assets/ba218455-70af-4184-ab95-45920d9d8e81)

If I look at `BETA_24`  then its last (merge) commit is **from last week**
![grafik](https://github.com/user-attachments/assets/58b4252f-f6d8-4b75-8343-e1e08c33617f)

and this PR contains commits from **2 week ago**

![grafik](https://github.com/user-attachments/assets/c83309bd-be10-4b1e-a4ba-2590270953be)

so maybe _this PR_ just needs to be rebased on top of `BETA_24` to fix the issue...

Reply:
> You can try this if you just don't want to see the error ...

It seems skipping baseline checks is indeed our best option for now, simply because we have no indication what causes the check to check files which are marked to be ignored.

> For example if I look here it says the beta-branch is behind master (but that might be intended I don't know):

It's a long running branch, which means that is does not and cannot immediately reflect all changes in master.

> and this PR contains commits from 2 week ago

Right, but when those commits were freshly pushed 2 weeks ago they caused the same build failure. I think its the very nature of a commit from 2 weeks a ago that it was created 2 weeks ago, no?

> so maybe this PR just needs to be rebased on top of BETA_24 to fix the issue...

Just today I did exactly that rebase for #3390  and still after rebase we get the same bogus error. Ergo, we haven't understood the first thing about that error message.

Reply:
> Just today I did exactly that rebase for https://github.com/eclipse-jdt/eclipse.jdt.core/pull/3390 and still after rebase we get the same bogus error. Ergo, we haven't understood the first thing about that error message.

I have already explained the error message before and according of you problem [here](https://github.com/eclipse-jdt/eclipse.jdt.core/pull/3446) it then looks like BETA_24 needs to be synced with master.

Reply:
> sorry for being dense but I can't recall any statement from you that explained (in words which I can understand) why files META-INF/ECLIPSE_.RSA and META-INF/ECLIPSE_.SF should ever give a reason for build failure. Without any connection to the error message all the recommendations you may give are completely black box, voodoo. Nothing that will help me to improve the situation on my own, ever.

I elaborated the details on a recent example here:
- https://github.com/eclipse-jdt/eclipse.jdt.core/pull/3446#issuecomment-2547728849

Summary is given here:
- https://github.com/eclipse-platform/eclipse.platform.releng.aggregator/issues/2660#issuecomment-2547734985

if anything is still unclear please let me know.

Beside this its nothing voodo or only in my head but as platform builds are working for a long time (even before I started contributing to eclipse) so I can only take these "as-is" as other have decided over the details long before.